usage-examples/reference-data-sets/default.svg

```diff
@@ -0,0 +1,5448 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   xmlns:xhtml="http://www.w3.org/1999/xhtml"
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
```

Comment:
EDef1, EDef2 and ArtC3 are missing there event names (EnvironmentDefined and ArtifactCreated, respectively)
TCS3 exists in two events, the lower one should be TCS4